apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java

```diff
@@ -66,11 +67,18 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         if (allArguments[0] instanceof CommandsData) {
             operationName = operationName + "BATCH_EXECUTE";
             command = "BATCH_EXECUTE";
+            if (RedissonPluginConfig.Plugin.Redisson.SHOW_BATCH_COMMANDS) {
+                command += ":" + showBatchCommands((CommandsData) allArguments[0]);
+            }
         } else if (allArguments[0] instanceof CommandData) {
             CommandData commandData = (CommandData) allArguments[0];
             command = commandData.getCommand().getName();
-            operationName = operationName + command;
-            arguments = commandData.getParams();
+            if ("PING".equals(command) && !RedissonPluginConfig.Plugin.Redisson.SHOW_PING_COMMAND) {
```

Comment:
@wu-sheng This change will lead to a problem that the plugin cannot find an active span to close after "Ping" execution. Skywalking would throw a NullPointerException. Maybe it needs a flag to duel this situation.
![NullPointerException](https://github.com/apache/skywalking-java/assets/63052374/75bed105-a871-47c4-ae50-b800d1888c0c)

Reply:
Do you want to write that PR? I think this is easy. You just need to add `if ("PING".equals(command) && !RedissonPluginConfig.Plugin.Redisson.SHOW_PING_COMMAND) {` check in the `#afterMethod` and avoid `ContextManager.stopSpan();` in this case.

Reply:
@CzyerChen I think this is definitely going to happen.

Reply:
This issue has been resolved by @Ricehomesky from our team. I suppose he can raise a PR very soon.